### ISSUE TITLE
Add option to not block taskbar

### ISSUE THREAD

**Issue by xsrvmy** [Codeusa/Borderless-Gaming#545]
Is it possible to add an option to not have the app cover the taskbar (basically maximized with no title bar)? So that it is possible to see tray icons while playing (and clock on win 11).

**Comment by SystemParadox** [comment]
Yes please, I was really surprised this wasn't an option already. I get why many people want it hidden, but it also seems like a pretty common request, to keep the taskbar visible so you can treat the game like any other application and switch between things quickly.
